policyengine_canada/parameters/gov/provinces/ab/benefits/marital_status/base.yaml

```diff
@@ -0,0 +1,10 @@
+description: Alberta marital status base credit
+values:
+  2023-01-01: 19_369
```

Comment:
Can you find a more recent document to cite as well, as to confirm that it's not inflation-indexed?